packages/contracts-bedrock/src/dispute/interfaces/IDisputeGameFactory.sol

```diff
@@ -28,6 +28,9 @@ interface IDisputeGameFactory {
     struct GameSearchResult {
         uint256 index;
         GameId metadata;
+        Timestamp timestamp;
```

Comment:
The timestamp is packed inside of the `GameId`

Reply:
Ehhh honestly because it's annoying enough to pull out the timestamp and this is a view function that can be as expensive as we want, I prefer to just give the timestamp explicitly.